src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -116,12 +112,13 @@
             unsigned-transaction        [:wallet.send/unsigned-transaction]
             network                     [:get-current-account-network]
             {gas-edit       :gas
+             max-fee        :max-fee
              gas-price-edit :gas-price} [:wallet/edit]]
     (let [modal?         (:id send-transaction)
           ;;TODO(goranjovic) - unify unsigned and regular transaction subs
           {:keys [amount symbol] :as transaction} (if modal? unsigned-transaction send-transaction)
-          gas            (or (:value gas-edit) (:gas transaction))
-          gas-price      (or (:value gas-price-edit) (:gas-price transaction))
```

Comment:
Is the previous value initialized somewhere else (also is it looking both in `unsigned-transaction` and `send-transaction`? 
If not, it might not work for dapp transactions - when they create transactions in dapp code they may set those fields and in that case we need to show those values initially here.

Reply:
Not sure if I missed something, but I only see previous `*-edit` values being used, not any pre-existing gas settings. Maybe edit values are initialized somewhere else?

Reply:
@goranjovic yes, I have changed the subscription so that it (in the `subs` file), so that it will use the values from the transaction instead (I just moved that code to the subscription, that is ), so we can calculate `max-fee` there if necessary.

Reply:
actually, signed / unsigned transactions, let me check

Reply:
ah need to fix unsigned/signed, will have a look